core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1155,6 +1157,14 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
       .setName("projects/" + projectName)
       .setFilter("metric.type=\"storage.googleapis.com/storage/v2/total_bytes\"")
       .setInterval(interval)
```

Comment:
if you're aggregating to a full 24-hour period, would it make sense to increase the interval above 5 minutes to ensure data? I'm not sure if it is necessary; your PR description sounds like it is not.

Reply:
Yeah I was a little confused about combining the interval with the aggregation period.  Does the aggregation just disregard the interval or what?

Reply:
My understanding - which could be wrong - is that interval is processed first, and Google finds all the time series within that interval, which is every 5 minutes. Then, it takes those every-5-minute data points and aggregates them into every-`aggregation.alignmentPeriod` data points.

Reply:
https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.alertPolicies#Aggregation is what I was looking at

Reply:
That sounds reasonable, but not sure it explains why the aggregation would work when it doesn't without it?  Since there wouldn't be any data points to aggregate.  Overall I'm still very confused

Reply:
Ok, further experimentation suggests that it takes only the END time of the given interval, and then aggregates the previous alignmentPeriod up to that time.  So if you give it the interval 2:30-2:35, it'll take everything in the 24 hours from 2:30 previous day to 2:30 day of the interval.  If you give it an interval greater than the alignmentPeriod, say M2:00-T2:30, then you'll get one point for M2:30-T2:30 and one for M2:00-M2:30.  So I think the way I have it written now is FINE